src/jobservice/job/impl/gc/garbage_collection.go

```diff
@@ -415,8 +416,11 @@ func (gc *GarbageCollector) deletedArt(ctx job.Context) (map[string][]model.Arti
 	return artMap, nil
 }
 
-// clean the untagged blobs in each project, these blobs are not referenced by any manifest and will be cleaned by GC
-func (gc *GarbageCollector) removeUntaggedBlobs(ctx job.Context) {
+// mark or sweep the untagged blobs in each project, these blobs are not referenced by any manifest and will be cleaned by GC
+// * dry-run, find and return the untagged blobs
+// * non dry-run, remove the untagged blobs
```

Comment:
This is not accurate cause just remove the reference between blob and project, it's not removal.